libamqpprox/amqpprox_sessionstate.cpp

```diff
@@ -54,7 +54,7 @@ SessionState::SessionState(
 {
```

Comment:
I think d_readyToConnectOnUnpause should be initialised during construction? Might need to change the order here. But not able to comment above this line.
```suggestion
, d_readyToConnectOnUnpause(false)
{
```

Reply:
This was causing the test failure - thanks!